libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -14,6 +14,16 @@ AP_Camera_Backend::AP_Camera_Backend(AP_Camera &frontend, AP_Camera_Params &para
     _instance(instance)
 {}
 
+// init - performs any required initialisation
+void AP_Camera_Backend::init() {
```

Comment:
```suggestion
void AP_Camera_Backend::init()
{
```

similarly elsewere throughout this file